time-series/time-series-api/src/main/java/com/powsybl/timeseries/ast/NodeCalcVisitors.java

```diff
@@ -68,25 +68,26 @@ private NodeCalcVisitors() {
             NodeWrapper nodeWrapper) {
         if (nodeWrapper.node != null) {
             List<NodeCalc> children = nodeWrapper.node.acceptIterate(visitor, arg);
-            Collections.reverse(children);
-            for (NodeCalc child : children) {
-                visitQueue.push(new NodeWrapper(child, true));
+            int size = children.size();
+            for (int i = size - 1; i >= 0; i--) {
+                visitQueue.push(new NodeWrapper(children.get(i)));
```

Comment:
Since the code is readable enough like this, and since I don't expect Lists.reverse to have superior performance to the raw index manipulation, I would prefer to keep it like this. Is it ok ?